core/templates/pages/topic-viewer-page/stories-list/topic-viewer-stories-list.component.html

```diff
@@ -3,7 +3,7 @@
 
 <div class="stories-list">
   <div class="oppia-stories-list-container">
-    <mat-card class="oppia-page-card oppia-long-text" [ngClass]="checkTabletView() ? 'm-0': 'mat-card-margin-desktop'">
```

Comment:
Why was this removed?

Reply:
when we are changing from tablet to mobile view, this m-0 is still present. which cause heading (like `place values` given in proof vedio) to move upward and ultimately content is not visible. because it content is shifted behind nav tabs. so to fix this i removed it and with some css changes issue is fixed!!

Reply:
Hm, I'm a bit confused, did you mean that there is a scenario where 'm-0' and 'mat-card-margin-desktop' is present in the class attribute together?

Reply:
No, I mean to say `checkTabletView` only checks for width less than 768px and whenever this condition is satisfied `m-0` is replaced with `mat-card-margin-desktop`. So, whenever we hit mobile view (which is less than 500px) this condition is still valid, so `m-0` is still present as a class which is not an intended behaviour and this causes content to move up (behind the nav).

Reply:
Thanks for explaining! Just to confirm, does the "tablet view" work as expected after your changes?

Reply:
Previously this lessons section looks odd for bigger tablet screens, but it is fixed by changing to margins.

current behavior on oppia.org:-

https://user-images.githubusercontent.com/12553576/157441073-3d4d5830-8ae1-4ff4-995d-97fdffe33545.mp4

On test server this looks more terrible, for this refer to issue!!